services/minespace-web/src/components/common/CoreTableCommonColumns.tsx

```diff
@@ -111,7 +112,7 @@ export const renderActionsColumn = (
       return (
         <div>
           {items.length > 0 && (
-            <Dropdown menu={{ items }} placement="bottomLeft">
+            <Dropdown menu={{ items }} placement="bottomLeft" disabled={isRowSelected}>
```

Comment:
We could probably extract a function from the bit above (the stuff that happens in filteredActions.map() to create the items) so that we can reduce some of the bulk in the DocumentTable, the bulkItems seem to be the same.

Reply:
I left this one as discussed in our call.